infoschema/builder.go

```diff
@@ -98,6 +103,19 @@ func (b *Builder) ApplyDiff(m *meta.Meta, diff *model.SchemaDiff) ([]int64, erro
 	return tblIDs, nil
 }
 
+func (b *Builder) getAllocatorAfterRenameTable(m *meta.Meta, oldSchemaID, newSchemaID, tableID int64) (autoid.Allocator, error) {
+	id, err := m.GetAutoTableID(oldSchemaID, tableID)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	alloc := autoid.NewAllocator(b.handle.store, newSchemaID)
+	err = alloc.Rebase(tableID, id, true)
```

Comment:
What about we have multiple tidb-servers?

Reply:
We use `GetAutoTableID` to get the ID from TiKV, so it‘s no problem.

Reply:
Move this to the ddl owner.